stravalib/model.py

```diff
@@ -259,6 +409,21 @@ class Club(
     BackwardCompatibilityMixin,
     BoundClientEntity,
 ):
+    """
+    Represents a single club with detailed information about the club including
+    club name, id, location, activity types, etc.
+
+    See Also
```

Comment:
i started to do this for docstrings but am wondering if we need it as we'd use it a lot?